src/EPR.Calculator.Frontend/Constants/ViewNames.cs

```diff
@@ -16,5 +16,6 @@ public static class ViewNames
         public const string UploadFileIndex = "~/Views/UploadFile/Index.cshtml";
         public const string UploadFileProcessingIndex = "~/Views/UploadFileProcessing/Index.cshtml";
         public const string UploadFileRefresh = "~/Views/UploadFile/Refresh.cshtml";
+        public const string Empty = "";
```

Comment:
You Dnt need a constant for this
You can use string.Empty

Reply:
i have tried but it was failing the test as null reference

Reply:
but done that

Reply:
Can we debug in a call? Not sure what you mean. Thanks

Reply:
done it